Dockerfile

```diff
@@ -80,20 +89,15 @@ ENV PATH=${cmake_path}:$PATH
 
 COPY scripts scripts
 
-COPY ./irods-dev_4.3.0-1~focal_amd64.deb ./irods-runtime_4.3.0-1~focal_amd64.deb /irods43X_packages/
-
-RUN bash scripts/get_build_deps.sh
-
 RUN mkdir irods_s3_bridge
 
 COPY . irods_s3_bridge
```

Comment:
can we remove the word bridge from everything?

irods_s3_api?

Reply:
Sure, but that could block PR #33.

Probably better to do the rename after all of the PRs are merged?

Reply:
Certainly.